### ISSUE TITLE
📝 Rule Change: `padded-blocks` - Option to set minimum number of statements to require in block, before applying the rule

### ISSUE THREAD

**Issue by flosker** [eslint/eslint#15124]
### What rule do you want to change?

padded-blocks

### What change to do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Config

```js
"padded-blocks": [
      "warn",
      "always"
]
```

### Example code
```js
// 1
if (true) {
	console.log('Hello!')
}

// 2
if (true) {
	console.log('Hello!')
	console.log('Hello 2!')
	console.log('Hello 3!')
}
```

### What does the rule currently do for this code?

It requires padding for all blocks. Like below. Which looks ugly for blocks with very few statements.

```
if (true) {

	console.log('Hello!')

}
```

### What will the rule do after it's changed?

User can set option of min. number of statements to have in block, before applying this rule.

So with min. number set to `3`, 1st code block above will be ignored & won't produce errors. Same cannot be said for the 2nd one however.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by nzakas** [comment]
Thanks for the input. As [stated in the README](https://github.com/eslint/eslint#stylistic-rule-updates), we actually aren't making any further updates to rules that enforce stylistic preferences. If you have a strong preference for such a change, you can always copy the existing rule into a custom rule and make it behave the way you want.
